code/Test_definitions/population-density-data.feature

```diff
@@ -69,20 +76,24 @@ Feature: CAMARA Population Density Data API, v0.1.1
         And the response property "$.status" value is "AREA_NOT_SUPPORTED"
         And the response property "$.timedPopulationDensityData" is an empty array
 
-    @population_density_data_04_webhook_success_scenario
-    Scenario: Validate success response for a request specifying a webhook
+    @population_density_data_retrievePopulationDensity_04_async_success_scenario
+    Scenario: Validate success async response for a request when sink is provided
         Given the request body property "$.area" is set to a valid testing area within supported regions
         And the request body property "$.startDate" is set to a valid testing future date
         And the request body property "$.endDate" is set to a valid testing future date later than body property "$.startDate"
-        And the request body property "$.webhook.notificationUrl" is set to a valid https callback address
-        When the request "retrievePopulationDensity" is sent
+        And the request property "$.sink" is set to a URL when events can be monitored
+        And the request property "$.sinkCredentials.credentialType" is set to "ACCESSTOKEN"
+        And the request property "$.sinkCredentials.accessTokenType" is set to "bearer"
+        And the request property "$.sinkCredentials.accessToken" is set to a valid access token accepted by the events receiver
+        And the request property "$.sinkCredentials.accessTokenExpiresUtc" is set to a value long enough in the future        When the request "retrievePopulationDensity" is sent
```

Comment:
Typo here - When the request....must be on the next line

Reply:
solved